src/lib.rs

```diff
@@ -184,7 +184,7 @@ use std::collections::HashMap;
 pub const VERSION_DRAFT17: u32 = 0xff00_0011;
 
 /// The maximum length of a connection ID.
-pub const MAX_CONN_ID_LEN: usize = 18;
+pub const MAX_CONN_ID_LEN: u8 = 18;
```

Comment:
The public const should remain `usize` though, so it can be used for sizing arrays without having to convert it.